src/components/Documentation/Markdown/styles.js

```diff
@@ -31,6 +31,131 @@ export const Content = styled.article`
     animation-duration: 1s;
     animation-fill-mode: both;
     animation-name: fadeIn;
+
+    a[target='_blank']:after {
+      position: relative;
+      top: 1px;
+      right: 0;
+      width: 12px;
+      height: 12px;
+      margin-left: 1px;
+      content: url(/img/external-link.svg);
+    }
+
+    pre[class*='language-'] {
```

Comment:
we will need to share it anyway? move it to a separate file?

Reply:
We will. but we will first convert `styled-components` to something else. If we move it right now we will still need to rewrite it later so I leave it this way for now.

Reply:
kk, resolving 👍